rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -520,11 +520,15 @@ def train(
         possible_to_train = self._check_enough_labels(session_data)
 
         if not possible_to_train:
-            logger.error(
-                "Can not train a classifier. "
-                "Need at least 2 different classes. "
-                "Skipping training of classifier."
-            )
+            # logger.error(
+            #     "Can not train a classifier. "
+            #     "Need at least 2 different classes. "
+            #     "Skipping training of classifier."
+            # )
+            raise Exception("Can not train a classifier. "
```

Comment:
Maybe this is not the best solution. Because you can train your chatbot even without a single intent. Looking on other `raise` examples in this class, appears to be used only if the "problem" is big enough to disable the training. Is there anything that I am missing?